content/rs/getting-started/getting-started-crdbs.md

```diff
@@ -75,6 +75,8 @@ Repeat the same operations for cluster 2 with these differences:
 Now we have two Redis Enterprise Software clusters with FQDNs
 **cluster1.local** and **cluster2.local**.
 
+Important Note: The cluster must be able to fully resolve DNS of each of the clusters and the databases that it is connecting to. Without this resolution the cluster will fail to connect successfully.
```

Comment:
I didn't get the "databases" point. Why the databases are relevant here?

Reply:
for instance one of the things I had a problem with was ensuring that not only the cluster name resolved, but also that the DB name resolved. For instance it's not sufficient that cluster.local resolves but that redis-12000.cluster.local resolves. Otherwise, connections will fail.

Reply:
The cluster resolving is the responsibility of the customer, the db resolution is kind of ours...

Reply:
@Paz-Yanover - I guess I'm confused if this is a bug or not. But that said the primary objective was the document the ports to use the encrypted port. Should we remove this section and move to change the port numbers?

Reply:
Sounds good to me

Reply:
Just removed. @Paz-Yanover @bmansheim  anything else from your perspective?

Reply:
Fine with me